src/test/java/restel/core/parser/dto/TestWrappersTest.java

```diff
@@ -0,0 +1,33 @@
+package restel.core.parser.dto;
+
+import com.techconative.restel.core.parser.dto.TestApiWrappers;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TestWrappersTest {
+
+  @Test
+  public void testTestDefinitions() {
```

Comment:
What's the intention behind the test? And how is it different from what is intended for `RestelTestWrapperTest# RestelTestWrapperTest`?

Reply:
Test file removed